lib/components/dashboard/degree-info/distributionFunctions.tsx

```diff
@@ -66,9 +66,13 @@ export const getBoolExpr = (
     } else {
       concat = handleTagType(splitArr, index, course);
     }
+    if (index === 0 && concat === '&&!') {
+      boolExpr = boolExpr.concat('true');
+    }
     if (concat.length > 3) {
       index = index + 2;
     } else index++;
+
     boolExpr = boolExpr.concat(concat); // Causing issues with biology major.
```

Comment:
Is this comment still relevant?

Reply:
Not sure, do you remember why this was an issue? I did some testing on Mol Cell Bio major and didn't find any issues.

Reply:
Awesome! Let's assume this is fixed! Pls remove the comment and commit that ^.^